charts/matrix-neoboard-widget/templates/configmap.yaml

```diff
@@ -1,8 +1,19 @@
-apiVersion: v1
-kind: ConfigMap
+{{/*
+SPDX-FileCopyrightText: 2023 Nordeck IT & Consulting GmbH"
+SPDX-License-Identifier: Apache-2.0
+*/}}
+
+kind: "ConfigMap"
+apiVersion: "v1"
 metadata:
-  name: {{ include "app.fullname" . }}-cm
+  name: "{{ include "common.names.fullname" . }}"
   labels:
-{{ include "app.labels" . | indent 4 }}
+    {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
 data:
-  REACT_APP_HOME_SERVER_URL: "https://synapse.{{- .Values.settings.hostname -}}"
+  REACT_APP_HOME_SERVER_URL: "https://synapse.{{- .Values.global.domain -}}"
```

Comment:
Instead of `synapse`, we can use the `.Values.global.subdomain.synapse` configuration.